energyplus_regressions/diffs/table_diff.py

```diff
@@ -36,6 +34,7 @@
 __copyright__ = "Copyright (c) 2009 Santosh Philip and Amir Roth 2013"
 __license__ = "GNU General Public License Version 3"
 
+from pathlib import Path
```

Comment:
Start modernizing slightly using Path instances.  Could be done lots more places, but I didn't want to grow scope toooo much.